packages/turf-grid-to-matrix/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@turf/grid-to-matrix",
+  "version": "4.1.0",
+  "description": "turf grid-to-matrix module",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "scripts": {
+    "test": "node test.js",
+    "bench": "node bench.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Turfjs/turf.git"
+  },
+  "keywords": [
+    "turf",
+    "grid",
+    "points",
+    "array",
+    "matrix"
+  ],
+  "author": "Turf Authors",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Turfjs/turf/issues"
+  },
+  "homepage": "https://github.com/Turfjs/turf",
+  "devDependencies": {
+    "@turf/meta": "^4.1.0",
+    "@turf/truncate": "^4.1.0",
+    "@turf/point-grid": "^4.1.0",
+    "benchmark": "^2.1.3",
+    "load-json-file": "^2.0.0",
+    "tape": "^4.6.3",
+    "write-json-file": "^2.0.0"
+  },
+  "dependencies": {
```

Comment:
I think `@turf/meta` is missing and you've accidently added bbox, distance and helpers